libevmasm/GasMeter.h

```diff
@@ -47,37 +47,71 @@ namespace GasCosts
 	static unsigned const tier5Gas = 10;
 	static unsigned const tier6Gas = 20;
 	static unsigned const tier7Gas = 0;
-	inline unsigned extCodeGas(langutil::EVMVersion _evmVersion)
-	{
-		return _evmVersion >= langutil::EVMVersion::tangerineWhistle() ? 700 : 20;
-	}
-	inline unsigned balanceGas(langutil::EVMVersion _evmVersion)
-	{
-		if (_evmVersion >= langutil::EVMVersion::istanbul())
-			return 700;
-		else if (_evmVersion >= langutil::EVMVersion::tangerineWhistle())
-			return 400;
-		else
-			return 20;
-	}
 	static unsigned const expGas = 10;
 	inline unsigned expByteGas(langutil::EVMVersion _evmVersion)
 	{
 		return _evmVersion >= langutil::EVMVersion::spuriousDragon() ? 50 : 10;
 	}
 	static unsigned const keccak256Gas = 30;
 	static unsigned const keccak256WordGas = 6;
+	/// Corresponds to COLD_SLOAD_COST from EIP 2929
```

Comment:
Nit: EIPs are referred to as `EIP-2929` (with the dash).

Reply:
Fixed.